### PR TITLE
ssi_all: add "preserve_original_event" tag to documents with event.kind set to "pipeline_error"

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

```
ssi_all: add "preserve_original_event" tag to documents with event.kind set to "pipeline_error"

Omit problematic packages: google_workspace, jamf_protect and ti_mandiant_advantage.

[git-generate]
for f in $(
    (
        for p in $(
            yq 'select(.owner.github == "elastic/security-service-integrations")|.name' packages/**/manifest.yml \
            | grep -v -- ---
        ); do
            rg -l -g 'default.yml' "value: pipeline_error" packages/$p
        done
    )|sort|uniq|egrep -v 'google_workspace|jamf_protect|ti_mandiant_advantage'
); do
    (grep 'value: preserve_original_event' $f >/dev/null 2>&1) && continue
    perl -i -pe 'BEGIN{undef $/;} s/([a-z:"])
( *)(- set:.*value: pipeline_error)/$1
$2$3
$2- append:
$2    field: tags
$2    value: preserve_original_event
$2    allow_duplicates: false/smg' $f
done
for p in $(git diff --name-only HEAD~1|cut -d/ -f1,2|sort|uniq); do
    (
        cd $p
        elastic-package changelog add \
            --description 'Add "preserve_original_event" tag to documents with `event.kind` set to "pipeline_error".' \
            --type enhancement \
            --next minor \
            --link https://github.com/elastic/integrations/pull/12046
    )>/dev/null 2>&1
done
```

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [ ] 

## Related issues

- For #12045
- For #12067

### CODE REVIEW

**Comment by andrewkroh** [comment]
I think we will want `allow_duplicates: false` on that append processor.

File: packages/jamf_protect/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.7.0"
```
Comment (kcreddy) [inline packages/jamf_protect/changelog.yml:2]: This one doesn't seem to have any data-streams modified.

Comment (kcreddy) [reply packages/jamf_protect/changelog.yml:2]: Looks like it is in excluded list. changelog entry to be removed.

**Review by kcreddy — APPROVED** [review]
Minor change. LGTM otherwise 👍🏼

File: packages/jamf_protect/changelog.yml (hunk 1 above)
Comment (efd6) [reply packages/jamf_protect/changelog.yml]: Also, azure. Also remove.

**Review by ishleenk17 — APPROVED** [review]
Only AWS bedrock for infraobs ownership. Change looks good for that.

**Review by shmsr — APPROVED** [review]

**Pull request merged**